metrics/metrics.go

```diff
@@ -78,72 +69,38 @@ var (
 		TagKeys:     []tag.Key{Direction, UseFD, Op},
 	}
 
-	RcmgrAllowConnView = &view.View{
-		Measure:     RcmgrAllowConn,
+	RcmgrPeerView = &view.View{
+		Measure:     RcmgrPeer,
 		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{Direction, UseFD},
+		TagKeys:     []tag.Key{PeerID, Op},
```

Comment:
```suggestion
		TagKeys:     []tag.Key{Op},
```
Monitoring of peer_id stuff needs to be done by logging not prometheus metrics.

Reply:
resolved on the latest commit.